loco/hello-world/README.md

```diff
@@ -0,0 +1,16 @@
+# Shuttle Loco template :train:
+
+Welcome to the [Loco](https://loco.rs) Hello World Shuttle Template, designed with a focus on controllers and views (response schema). This minimalistic template comes with several features to kickstart your project:
+- **REST API Service without a Database**
+- **Workers**: Full-blown background job processing
+- **Task**: Ad-hoc functionalities to handle specific aspects of your application
+- **Tests**: Testing facility and examples
+- And More Loco Features
+
+## Quick Start
+
+To launch your app, simply use the following command:
+
+```sh
+$ cargo loco start
```

Comment:
The preferred way when Shuttle has been added would be 
```suggestion
cargo shuttle run
```

Reply:
changed